monitor/cli.py

```diff
@@ -247,27 +237,17 @@ def report(ctx, days_back, config_dir, profiles_dir, update_dbt_package, profile
     help="Force downloading the latest version of the edr internal dbt package (usually this is not needed, "
          "see documentation to learn more)."
 )
-@click.option(
-    '--slack-webhook', '-s',
-    type=str,
-    default=None,
-    help="A slack webhook URL for sending alerts to a specific channel (also could be configured once in config.yml)"
-)
 @click.option(
     '--slack-token', '-st',
     type=str,
     default=None,
     help="A slack token for sending alerts over slack (also could be configured once in config.yml)",
-    cls=RequiredIf,
```

Comment:
Don't we want the require if 🤔 ?
It lets users to know right in the start that they must provide both

Reply:
No because it doesn't allow users to mix.
For instance, I wanted to have my token in the config file but I wanted to be able to choose the channel easily within the CLI rather than edit the config file.